lib/ret_web/channels/hub_channel.ex

```diff
@@ -74,12 +95,21 @@ defmodule RetWeb.HubChannel do
     :ok
   end
 
+  defp broadcast_presence_update(socket) do
+    Presence.update(socket, socket.assigns.session_id, socket |> presence_meta_for_socket)
+    socket
+  end
+
+  defp presence_meta_for_socket(socket) do
+    socket.assigns |> Map.take([:hub_id, :presence, :profile, :context])
+  end
+
   defp join_with_hub(%Hub{entry_mode: :deny}, _socket) do
     {:error, %{message: "Hub no longer accessible", reason: "closed"}}
   end
 
   defp join_with_hub(%Hub{} = hub, socket) do
-    with socket <- assign(socket, :hub_sid, hub.hub_sid),
+    with socket <- socket |> assign(:hub_sid, hub.hub_sid) |> assign(:presence, :lobby),
```

Comment:
Shouldn't there be a presence broadcast on join? Or is that incidentally handleded by `events:profile_updated`?